types/jasmine-data_driven_tests/index.d.ts

```diff
@@ -34,6 +34,6 @@ interface JasmineDataDrivenTest {
         assertion: (arg0: T, arg1: U, done: () => void) => void): void;
     <T>(
         description: string,
-        dataset: T[],
+        dataset: T[] | Array<[T]>,
```

Comment:
Out of curiosity, why mix the `x[]` and `Array<x>` syntaxes?

Reply:
first thing that came into my mind, when I was implementing this, was to make it consistent with previous lines; however, there `Array<[T, U]>` has more sense than `Array<[T]>` has here. I tried to change it to `T[][]` (in https://github.com/DefinitelyTyped/DefinitelyTyped/commit/60cc2cdbbb0b831eae544b34acbec7c9a723578a), which - AFAIK - is the same, however tslint failed when I've done this; I really don't know why :(

Reply:
Interesting.  I'm not sure why the linter would flag that.  In any case, it's fine the way it is.  Thanks for following up!